test/functional/cypress/integration/ecs/server_group.spec.js

```diff
@@ -0,0 +1,256 @@
+
+    cy.get('task-definition-react .Select-placeholder:contains("Select a target group")').type('demo');
+
+    cy.get('.Select-option:contains("demo")').click();
+
+    cy.get('div[ng-model="$ctrl.command.launchType"]').type('FARGATE');
+
+    cy.get('.ui-select-highlight:contains("FARGATE")').click();
+
+    cy.get('submit-button[label="command.viewState.submitButtonLabel"]').click();
+
+    cy.get('.account-tag').should('have.length', 2);
+
+    cy.get('td:contains("ecsapp-prod-ecsdemo")').should('have.length', 1);
+    cy.get('td:contains("ecsapp-functional-testing")').should('have.length', 1);
+
+    cy.get('td:contains("us-west-2")').should('have.length', 2);
+
+    cy.get('button[ng-click="pipelineConfigurerCtrl.revertPipelineChanges()"]').click();
+  });
+
+  it('configure a new server group with container inputs', () => {
+    cy.visit('#/applications/ecsapp/executions');
+
+    cy.get('a:contains("Configure")').click({ force: true });
+
+    cy.get('a:contains("Deploy")').click({ force: true });
+
+    cy.get('.btn:contains("Add server group")').click();
+
+    cy.get('span:contains("Continue")').click();
+
+    cy.get('div[ng-model="command.ecsClusterName"]').type('spinnaker-deployment-cluster');
+
+    cy.get('span:contains("spinnaker-deployment-cluster")').click();
+
+    cy.get('input[ng-model="command.stack"]').type('functional');
+
+    cy.get('input[ng-model="command.freeFormDetails"]').type('testing');
+
+    cy.get('div[ng-model="$ctrl.command.networkMode"]').type('awsvpc');
+
+    cy.get('span:contains("awsvpc")').click();
+
+    cy.get('div[ng-model="$ctrl.command.subnetType"]').type('public');
+
+    cy.get('span:contains("public-subnet")').click();
+
+    cy.get('input[ng-model="$ctrl.command.associatePublicIpAddress"]')
+      .eq(1)
+      .click();
+
+    cy.get('.Select-placeholder:contains("Select an image")').type('TRIGGER');
+
+    cy.get('.Select-option:contains("TRIGGER")').click();
+
+    cy.get('container-react input[type=number]')
+      .eq(0)
+      .type(1024);
+
+    cy.get('container-react input[type=number]')
+      .eq(1)
+      .type(1024);
+
+    cy.get('div[ng-model="$ctrl.command.launchType"]').type('FARGATE');
+
+    cy.get('.ui-select-highlight:contains("FARGATE")').click();
+
+    cy.get('div[ng-model="$ctrl.command.logDriver"]').type('awslogs');
+
+    cy.get('span:contains("awslogs")').click();
+
+    cy.get('submit-button[label="command.viewState.submitButtonLabel"]').click();
+
+    cy.get('.account-tag').should('have.length', 2);
+
+    cy.get('td:contains("ecsapp-prod-ecsdemo")').should('have.length', 1);
+    cy.get('td:contains("ecsapp-functional-testing")').should('have.length', 1);
+
+    cy.get('td:contains("us-west-2")').should('have.length', 2);
+
+    cy.get('button[ng-click="pipelineConfigurerCtrl.revertPipelineChanges()"]').click();
+  });
+
+  it('edit an existing server group with artifact', () => {
```

Comment:
Since we're already exercising the inputs vs. artifact functionality of the modal in the "new SG" tests, what do you think about having just 1 "edit an existing server group..." test that's read-only (or almost so), to ensure it loads + displays the correct info? Would this omit some important codepaths?